thrift/thrift0.13/src/main/java/com/linecorp/armeria/server/thrift/THttpService.java

```diff
@@ -777,4 +819,40 @@ private HttpData encodeException(ServiceRequestContext ctx,
             }
         }
     }
+
+    private static final class DecodedRequest {
+
+        private final SerializationFormat serializationFormat;
+        private final int seqId;
+        private final ThriftFunction thriftFunction;
+        private final RpcRequest decodedReq;
+
+        private DecodedRequest(SerializationFormat serializationFormat, int seqId,
+                               ThriftFunction thriftFunction, RpcRequest decodedReq) {
+            this.serializationFormat = serializationFormat;
+            this.seqId = seqId;
+            this.thriftFunction = thriftFunction;
+            this.decodedReq = decodedReq;
+        }
+
+        @Override
+        public boolean equals(@Nullable Object obj) {
+            if (this == obj) {
+                return true;
+            }
+            if (!(obj instanceof DecodedRequest)) {
+                return false;
+            }
+            final DecodedRequest that = (DecodedRequest) obj;
+            return seqId == that.seqId &&
+                   Objects.equals(serializationFormat, that.serializationFormat) &&
+                   Objects.equals(thriftFunction, that.thriftFunction) &&
+                   Objects.equals(decodedReq, that.decodedReq);
+        }
+
+        @Override
+        public int hashCode() {
+            return Objects.hash(serializationFormat, seqId, thriftFunction, decodedReq);
+        }
```

Comment:
nit: These methods are unnecessary as there is no requirement for an equality check.

Reply:
I removed these methods!